sn_cli/src/subcommands/register.rs

```diff
@@ -101,11 +101,22 @@ async fn get_registers(names: Vec<String>, client: &Client) -> Result<()> {
         println!("Trying to retrieve Register from {xorname:?}, {tag}");
 
         match client.get_register(xorname, tag).await {
-            Ok(register) => println!(
-                "Successfully retrieved Register '{name}' from {}, {}!",
-                register.name(),
-                register.tag()
-            ),
+            Ok(register) => {
+                println!(
+                    "Successfully retrieved Register '{name}' from {}, {}!",
+                    register.name(),
+                    register.tag()
+                );
+                let entries = register.read();
```

Comment:
better get `Register` with its `Debug` fmt refactored to giving those info ? if they are interested.
also, printly out all bytes might be not necessory, just first couple of hex will be enough and more readable?
like what we do for XorName Debug?

Reply:
Added a `address()` method to `ClientRegister`, be carefull the `register` here is a `ClientRegister` (includes a `Client`), not a `Register`